pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>com.backbase.stream</groupId>
     <artifactId>stream-services</artifactId>
-    <version>3.37.1</version>
+    <version>3.38.0-SNAPSHOT</version>
```

Comment:
Why are you manually changing the version here?

Reply:
Why was no SNAPSHOT by default?

Reply:
We have a simplified development model, and the versions are bumped automatically, so there is no manual intervention required or SNAPSHOT versions, you can find more information here: https://github.com/Backbase/stream-services/blob/master/CONTRIBUTING.md

Reply:
that simplified model caused headaches for me, as I had to refer to a version in another project which is not available publicly.... so I had to bump the stream version to ensure mvn can not download (as refresh) the artifacts from the public repo.
I would suggest returning to the normal well-known and designed development model as the main/devel/whatever branch is SNAPSHOT and all development starts from there. (of course, I should not do manually, but was not there)
Shall I remove the version changes?
The linked page is not clear to me, sorry

Reply:
The current GH workflows do support the `develop` branch; we use it in specific cases where we are implementing a major release (it was used to implement Stream 3, for example), and in this case, we use the SNAPSHOT identifier; if you believe that is necessary for what you are trying to achieve now feel free to relieve it. By the way, it seems that it is long overdue to have this project taken over by some RnD team; we can discuss it anytime in the future.